frontend/src/pages/Home/components/Carousel/owl-carousel-styles.css

```diff
@@ -2,27 +2,39 @@
   position: absolute;
   top: 45%;
   transform: translateY(-50%);
-  height: 100px;
   background-color: #000;
   color: #fff;
   margin: 0;
   transition: all 0.3s ease-in-out;
+  height: 100px;
 }
 
-.owl-nav button.owl-prev {
-  background-color: #121f3a !important;
-  color: #fff !important;
+.owl-nav button.owl-prev,
+.owl-nav button.owl-next {
+  color: #111;
+  border-radius: 20px !important;
   padding: 0 15px !important;
-  left: 0;
 }
 
+.owl-nav button.owl-prev > span,
+.owl-nav button.owl-next > span {
+  transform: translateY(35%);
+}
+
+.owl-nav button.owl-prev:hover,
+.owl-nav button.owl-next:hover {
+  background-color: #333 !important;
```

Comment:
Instead of changing the background color on hover, Can we just change the array color maybe?

Reply:
okay i will try to inculcate these.

Reply:
i pushed the new and updated changes with new icons.